### PR TITLE
feat: added 'total claimed rewards' amount on 'Your Rewards' panel

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* added `total claimed rewards` amount on 'Your Rewards' panel
  * issue: https://github.com/AstarNetwork/astar-apps/issues/122   
* summarized circulating information (`current circulating` and `total circulating` supply) into one panel 
* added `holders` panel on the dashboard page
* refactor: use `wait` function instead of `setTimeout`

**Check list**
- [ ] contains breaking changes
- [x] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

**Adds**

image

image

### CODE REVIEW

File: src/components/dashboard/Dashboard.vue [hunk 1]
```diff
@@ -3,10 +3,10 @@
     <div class="container--chart-panels">
       <div class="container--value-panel">
         <div class="container--panel">
-          <value-panel title="Current Circulating Supply" :value="circulatingSupply" />
+          <circulating-panel :symbol="chainInfo.tokenSymbol" />
         </div>
         <div class="container--panel">
-          <value-panel title="Total Supply" :value="totalSupply" />
+          <rewards-panel :symbol="chainInfo.tokenSymbol" />
```
Comment (hoonsubin) [inline src/components/dashboard/Dashboard.vue:9]: if we plan to add this view on the dApps Staking page at some point, I don't think it makes sense to add this on the dashboard page now?
What are your thoughts @Kahonnohak ?

Comment (Kahonnohak) [reply src/components/dashboard/Dashboard.vue:9]: Yes this needs to be on dApp Staking. i just commented on Slack. The question is how much we are adding before we can move to V2. V1 just got "your rewards' section which is appropriate for showing the total earned rewards however the space is tight and will need additional frontend development which is pitty..) Let me think.

The total Earned display makes good encouragement to users indeed and is amazing, however, the ideal goal is to have the total earned per project not all-time.

Comment (impelcrypto) [reply src/components/dashboard/Dashboard.vue]: Fixed
image

**Comment by Kahonnohak** [comment]
Hi! 
1) I didn't get a live update for the claimed rewards amount after I claimed. Do we need to reload to be updated or can it be possible?

2) Please change the font-weight to 700 for XXX ASTR

**Comment by hoonsubin** [comment]
what is the status of this PR? @Kahonnohak @impelcrypto

**Review by hoonsubin — APPROVED** [review]

**Review by Kahonnohak — APPROVED** [review]
Thanks!

**Comment by impelcrypto** [comment]
@Kahonnohak 

>  I didn't get a live update for the claimed rewards amount after I claimed. Do we need to reload to be updated or can it be possible?
Please change the font-weight to 700 for XXX ASTR

Fixed :)
Demo:
https://www.loom.com/share/22b025c02a91424f910652022b537a58

**Pull request merged**